### ISSUE TITLE
Better default value for Binary Mode

### ISSUE THREAD

**Issue by vladiulianbogdan** [dougmoscrop/serverless-http#304]
I have a question regarding the [Binary Mode](https://github.com/dougmoscrop/serverless-http/blob/master/docs/ADVANCED.md#binary-mode). Could we set some better default values for binary? For example, `application/pdf` will just not work by default. The user has to set by itself the `BINARY_CONTENT_TYPES` env variable. 

If you agree that we could set some better defaults, I can come with a PR for this.

**Comment by dougmoscrop** [comment]
Yes that makes sense to me.  What do you think about going the inverse and having a list of known text types?
